src/passthrough/mod.rs

```diff
@@ -485,6 +661,12 @@ pub struct Config {
     /// * If dax_file_size == N, DAX will enable only when the file size is greater than or equal
     /// to N Bytes.
     pub dax_file_size: Option<u64>,
+
+    /// Whether to return host_inode instead of assigning a virtual inode
+    /// it should be noted that the returned host inode will be stored in the lower 47bits,
+    /// and for inodes over 47bits, virtual inode will be used instead.
+    /// The default value for this option is `false`.
+    pub use_host_ino: bool,
```

Comment:
Why do we need to control this behavior when it has known issues to return the host inode?

Reply:
The default implementation is the same as before, allocating a virtual inode at a time, but now when forgetting, it is not removed from memory to ensure consistency. The second way is to use host inode to encode, mainly because the internal hot upgrade will be more complicated under the two inode encoding methods, so the default implementation is retained.

Reply:
use_host_ino and libfuse's use_ino options are similar
https://manpages.debian.org/testing/fuse/mount.fuse.8.en.html#use_ino